docs/using-the-jdbc-driver/HostAvailabilityStrategy.md

```diff
@@ -0,0 +1,23 @@
+| Simple Host Availability Strategy              | `null` or blank      | This strategy is a simple pass through strategy. It does nothing when availability is set, and returns the raw host availability when fetched.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | 
+| Exponential Backoff Host Availability Strategy | `exponentialBackoff` | This strategy is passive by not actively health checking hosts. Instead it allows consumers of host availability check instead. <br/> If a host is marked `NOT_AVAILABLE`, there is a backoff time period where the hosts availability will return `NOT_AVAILABLE`. This backoff time period can be set using the `hostAvailabilityStrategyBaseBackoffTime` configuration parameter with the time units of seconds. After this period elapses, the hosts availability will return `AVAILABLE` so that consumers of this information may try the host out. <br/> Each subsequent time the host found to be down, the host is set to `NOT_AVAILABLE`, and another backoff period twice the duration of the last, will occur before the host availability starts to return `AVAILABLE`. If a host is found to be However, if the host is back up, consumers will set the host availability to `AVAILABLE` with no backoff period. The next backoff period that occurs will be reset to the base backoff time. | 
```

Comment:
```suggestion
| Exponential Backoff Host Availability Strategy | `exponentialBackoff` | This strategy does not actively health check hosts and instead allows host availability consumers to perform health checks. <br/><br/> If a host is marked `NOT_AVAILABLE`, there is a backoff time period where the host's availability will be `NOT_AVAILABLE`.  After the backoff period, the host's availability will be `AVAILABLE` so that host availability consumers can attempt to connect to the host. <br/><br/> Each subsequent time the host fails a health check, the host is set to `NOT_AVAILABLE` and another backoff period twice the duration of the last will occur before the host availability is set to `AVAILABLE` again. However, if the host passes the health check, consumers will set the host availability to `AVAILABLE` with no backoff period. The next backoff period that occurs will be reset to the base backoff time. <br/><br/> The backoff time period can be set in seconds using the `hostAvailabilityStrategyBaseBackoffTime` configuration parameter. | 
```
?

Reply:
Done!